_episodes/14-modules.md

```diff
@@ -147,6 +148,18 @@ required software dependencies.
 
 {% include {{ site.snippets }}/modules/software-dependencies.snip %}
 
+Note that this module loading process happens principally through
+the manipulation of environment variables like `$PATH`. There
```

Comment:
This maybe unclear to someone who has not used Linux/Unix very much. Perhaps one could examine the output of `echo $PATH` before and after loading a module to explain this better.

Reply:
For sure! This is examined a few lines above, inside of `{{ site.snippets }}/modules/python-module-path.snip %}`.